source/documentation/other-topics/persistent-volume-encryption-snapshot.html.md.erb

```diff
@@ -0,0 +1,128 @@
+---
+title: Persistent volume encryption at rest and daily snapshots
+last_reviewed_on: 2019-12-24
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+
+### Introduction
+
+This guide will give you an overview of how to set encryption at rest for the Persistent volumes and daily snapshots taken to back up Persistent volumes. 
+
+
+### Encrypting Persistent volumes
+
+For users who need encryption at rest enabled for persistence volumes, Cloud Platform team created [gp2-expand][gp2-expand] storge class. this `gp2-expand` [Storage Class][storge class] have set with parameter `encrypted: "true"`, which denotes the EBS volume should be encrypted.
+
+The name of a StorageClass object is significant, user should match the PersistentVolumeClaim to [gp2-expand][gp2-expand] Storage class, by setting the storageClassName attribute to the name of a `gp2-expand`, as shown in the example below. 
+
+
+```
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: myclaim
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 8Gi
+  storageClassName: gp2-expand
+  selector:
+    matchLabels:
+      release: "stable"
+    matchExpressions:
+      - {key: environment, operator: In, values: [dev]}
+
+```
+
+
+Using the above PersistentVolumeClaim example, the cluster will try to [dynamically provision][dynamic-provisioning] a [Persistent volume][persistent-volumes] specially for the PersistentVolumeClaim. This provisioning is based on [gp2-expand][gp2-expand] [Storage Class][storage-class-aws-ebs], which will encrypt the PersistentVolume using the [KMS][amazon-kms].
+
+
+### Snapshots to back up Persistent volumes
+
+Users may want to have snapshots taken to back up Persistent volumes, that can be done using the [snapshot lifecycle policy][aws_dlm_lifecycle_policy]. This example snapshot lifecycle policy below will create a snapshot of all tagged volumes, every 24 hours starting at 05:00 UTC. A maximum of 30 snapshots will be retained of a target volume. The oldest snapshot retained will be <= 1 months old. 
```

Comment:
This usage "Users may..." is a bit awkward. When *writing* the document, you might be thinking about "users", but the person *reading* the document is a person trying to get something done or learn about the topic. I suggest a more conversational tone. addressing the reader as "you".

Also, it's not necessary to caveat things with "may want to". If the user is reading the section that talks about backing up PVs, it's because they want to backup a PV!